app/src/main/java/io/neurolab/tools/Utils.java

```diff
@@ -0,0 +1,40 @@
+package io.neurolab.tools;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+
+public class Utils {
+
+    private static InputStream is;
+
+    public static int countLines(String filename) throws Exception {
+        return countLines(new File(filename));
+    }
+
+    public static int countLines(File file) throws Exception {
+
+        try (InputStream input = new BufferedInputStream(new FileInputStream(file))) {
+            is = input;
```

Comment:
Redundant variable

Reply:
Yes i had to use this so that i can use the InputStream object in the finally block. Will remove it now.